scripts/pylint_extensions_test.py

```diff
@@ -1985,6 +1985,7 @@ def setUp(self):
             pylint_extensions.SpaceBelowFileOverviewChecker)
         self.checker_test_object.setup_method()
 
+    # The checks assume that all files have a file overview.
```

Comment:
You need to explain why we are allowed to make this assumption.

I also suggest moving this to the class docstring. I.e. write a proper class docstring explaining what the class this, and then add this as a "Note: " within the docstring. Otherwise, this comment isn't really discoverable, and it seems like it only relates to the method just below it.

Reply:
done